botocore/utils.py

```diff
@@ -897,6 +973,11 @@ def deep_merge(base, extra):
         base[key] = extra[key]
 
 
+def translate_service_id(service_id):
```

Comment:
I would prefer we call this something like `get_service_event_name` as `translate_service_id` is a little vague and there are other ways to translate service id's. I see this as being similar to the `get_service_module_name` function.

Reply:
done